src/components/pages/History/PoolShares.tsx

```diff
@@ -1,86 +1,176 @@
-import { useMetadata, useOcean } from '@oceanprotocol/react'
+import { useOcean } from '@oceanprotocol/react'
 import React, { ReactElement, useEffect, useState } from 'react'
 import Table from '../../atoms/Table'
-import { DDO, Logger, MetadataCache } from '@oceanprotocol/lib'
-import PriceUnit from '../../atoms/Price/PriceUnit'
 import Conversion from '../../atoms/Price/Conversion'
 import styles from './PoolShares.module.css'
 import AssetTitle from '../../molecules/AssetListTitle'
+import { gql, useQuery } from '@apollo/client'
+import {
+  PoolShares as PoolSharesList,
+  PoolShares_poolShares as PoolShare,
+  PoolShares_poolShares_poolId_tokens as PoolSharePoolIdTokens
+} from '../../../@types/apollo/PoolShares'
+import web3 from 'web3'
+import Token from '../../organisms/AssetActions/Pool/Token'
+
+const poolSharesQuery = gql`
+  query PoolShares($user: String) {
+    poolShares(where: { userAddress: $user, balance_gt: 0.001 }, first: 1000) {
+      id
+      balance
+      userAddress {
+        id
+      }
+      poolId {
+        id
+        datatokenAddress
+        lockedValue
```

Comment:
ok, this was changed into `valueLocked`. This right now will only work against Rinkeby where latest subgraph is deployed

Reply:
right, this fixed it, thanks :)

Reply:
Also sometimes I don't get the Published assets on my history page

Reply:
99.5% sure this was the only change affecting us here. The deployment of latest subgraph to mainnet will be started now, and once that's finished this PR can be shipped

Reply:
aaaand it's deployed :-) So should now also work against mainnet

Reply:
I really see the need of a pagination for the published assets now that I have more of that

Reply:
it takes some time to sync, wait for it